devscripts/prepare_manpage.py

```diff
@@ -43,6 +43,27 @@ def filter_excluded_sections(readme):
         '', readme)
 
 
+def _convert_code_blocks(readme):
+    current_code_block = None
+
+    for line in readme.splitlines(True):
+        if current_code_block:
+            if line == f'{current_code_block}\n':
+                current_code_block = None
+                yield '\n'
+            else:
+                yield f'    {line}'
+        elif line.startswith('```'):
+            current_code_block = '`' * line.count('`')
```

Comment:
It might be better to not formatstring each iteration:
```suggestion
            if line == current_code_block:
                current_code_block = None
                yield '\n'
            else:
                yield f'    {line}'
        elif line.startswith('```'):
            current_code_block = line.count('`') * '`' + '\n'
```